main.go

```diff
@@ -182,18 +182,16 @@ func generateTargetArtifacts(w http.ResponseWriter, r *http.Request) {
 	name := mux.Vars(r)["name"]
 	plan := r.FormValue("plan")
 
-	helmTargetArtifactTypeValue := "Helm"
-	yamlsTargetArtifactTypeValue := "Yamls"
-	knativeTargetArtifactTypeValue := "Knative"
-
-	re := regexp.MustCompile(`artifacttype:\\s.*`)
-	artifactTypematch := re.FindString(plan)
-	artifactType := yamlsTargetArtifactTypeValue
-	if strings.HasSuffix(artifactTypematch, knativeTargetArtifactTypeValue) {
-		artifactType = knativeTargetArtifactTypeValue
-	} else if strings.HasSuffix(artifactTypematch, helmTargetArtifactTypeValue) {
-		artifactType = helmTargetArtifactTypeValue
+	re := regexp.MustCompile(`artifactType:\s*(.+)`)
+	artifactTypematch := re.FindStringSubmatch(plan)
+	var artifactType string
+	if len(artifactTypematch) == 0 {
+		artifactType = ""
+	} else {
+
+		artifactType = strings.ReplaceAll(artifactTypematch[1], " ", "")
 	}
+
```

Comment:
```suggestion
	if len(artifactTypematch) > 1 {
		artifactType = artifactTypematch[1]
	}
```

This can leverage the fact that the default value of golang string is "". Also since we are selecting only non-spaced characters we need not replace them. And since we are accessing the first element it is better to check for >1 as length.